models/public/densenet-161-tf/densenet-161-tf.md

```diff
@@ -0,0 +1,59 @@
+# densenet-161-tf
+
+## Use Case and High-Level Description
+
+This is an Tensorflow version of `densenet-161` model, one of the DenseNet
```

Comment:
You may enhance description with description of caffe models from [https://github.com/opencv/open_model_zoo/pull/174](https://github.com/opencv/open_model_zoo/pull/176/files)